tests/ansible_collections/roles/oracle-linux-specific/tasks/main.yml

```
@@ -1,2 +1,4 @@
 ---
 - include_tasks: boot_standard_kernel.yml
+# Workaround reference: https://issues.redhat.com/browse/RHELC-1180
+- include_tasks: stop_firewalld_workaround.yml
```

```suggestion
- include_tasks: stop_firewalld_workaround.yml
  when: distro == oracle-8
```
@kokesak, @danmyway, shouldn't we limit the workaround to OL8 only?

---

@bocekm yeah I thought it was done on a higher level here: https://github.com/oamg/convert2rhel/blob/main/tests/ansible_collections/main.yml#L14C13-L14C13 but this also apply for the oracle-7 which we do not want. I'll change it